packages/era/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "@lodestar/era",
+  "description": "Era file handling module for Lodestar",
+  "license": "Apache-2.0",
+  "author": "ChainSafe Systems",
+  "homepage": "https://github.com/ChainSafe/lodestar#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ChainSafe/lodestar.git"
+  },
+  "bugs": {
+    "url": "https://github.com/ChainSafe/lodestar/issues"
+  },
+  "version": "1.34.1",
+  "type": "module",
+  "exports": {
+    ".": {
+      "bun": "./src/index.ts",
+      "import": "./lib/index.js"
```

I think we need to add a `types` field to `exports["."]`
```suggestion
      "bun": "./src/index.ts",
      "types": "./lib/index.d.ts",
      "import": "./lib/index.js"
```

---

@guha-rahul please do another pass on this as mentioned earlier